modules/swagger-annotations/src/main/java/io/swagger/oas/annotations/media/Schema.java

```diff
@@ -39,192 +35,230 @@
+   * @return the schema's format
    **/
   String format() default "";
 
   /**
-   * 
+   * References a schema definition in an external OpenAPI document.
+   * @return a reference to this schema
    **/
   String ref() default "";
 
   /**
-   * if true, designates a value as possibly null
+   * If true, designates a value as possibly null.
+   * @return whether or not this schema is nullable
    **/
   boolean nullable() default false;
 
   /**
-   * sets whether the value can be modified by a consumer
+   * Sets whether the value should only be read during a response but not written to during a request.
+   * @return whether or not this schema is read only
    **/
   boolean readOnly() default false;
 
   /**
-   * sets whether a value can only be written to by a consumer
+   * Sets whether a value should only be written to during a request but not returned during a response.
+   * @return whether or not this schema is write only
    **/
   boolean writeOnly() default false;
 
   /**
-   * provides an array examples of the schema.  When associated with a specific media type, the example string shall be parsed by the consumer to be treated as an object or an array.
+   * Provides an example of the schema.  When associated with a specific media type, the example string shall be parsed by the consumer to be treated as an object or an array.
+   * @return an example of this schema
    **/
-  String[] examples() default "";
+  String example() default "";
 
   /**
-   * provides an example of the schema.  When associated with a specific media type, the example string shall be parsed by the consumer to be treated as an object or an array.
+   * Provides an array examples of the schema.  When associated with a specific media type, the example string shall be parsed by the consumer to be treated as an object or an array.
+   * @return a list of examples of this schema
    **/
-  String example() default "";
-
+  String[] examples() default {};
+  
   /**
-   * 
+   * Additional external documentation for this schema.
+   * @return additional schema documentation
    **/
   ExternalDocumentation externalDocs() default @ExternalDocumentation();
 
   /**
-   * 
+   * Specifies that a schema is deprecated and should be transitioned out of usage.
+   * @return whether or not this schema is deprecated
    **/
   boolean deprecated() default false;
 
   /**
-   * provides an override for the basic type of the schema.  Must be a valid type per the OpenAPI Specification
+   * Provides an override for the basic type of the schema.  Must be a valid type per the OpenAPI Specification.
+   * @return the type of this schema
    **/
   String type() default "";
 
   /**
-   * provides a list of allowable values.
+   * Provides a list of allowable values.  This field map to the enum property in the OAS schema.
+   * @return a list of allowed schema values
    */
-  String[] _enum() default {""};
+  String[] allowableValues() default {};
```

Comment:
To make it more natural for Java developers.  Starting with underscore brings a connotation of something being private.   I believe the name `allowableValues` is pretty clear on its intention, and I also updated the javadoc to say `This field map to the enum property in the OAS schema.`